src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -52,19 +52,19 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
             }
             toggled={scenario?.dailyReports}
             label="Daily Reports"
-            labelHelp="Tooltip help Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+            labelHelp="If enabled, your baseline scenario will be shared with Recidiviz and CSG. This data will only be used to provide you with daily reports."
```

Comment:
Unless "CSG" is an acronym that's well understood within this domain, you may want to consider spelling it out

Reply:
Fortunately it is!